pkg/client/auth/kerberos/spnego_windows.go

```diff
@@ -3,7 +3,10 @@ package kerberos
 import (
```

Comment:
these changes seem unrelated to the pr

Reply:
The errcheck linter complained about these.
I see now that pkg is excluded from the errcheck running in CI. I can move these to a separate PR.